yarn.lock

```diff
@@ -9920,6 +9920,15 @@ passport-oauth2@1.x.x:
     uid2 "0.0.x"
     utils-merge "1.x.x"
 
+passport-oauth@0.1.x:
```

Comment:
Looks like we're adding some old packages. Maybe we want to maintain our own Strava lib or some updated one. 

The one we use is 6+ years old: https://github.com/millsy/passport-strava